package-lock.json

```diff
@@ -1,9726 +1,8 @@
 {
   "name": "@microsoft/powerplatform-actions",
   "version": "0.1.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
please update your local npm client to latest (your client is likely still on version 6 -> ```npm --version```):
npm install -g npm

then best delete the package-lock.json locally and rerun ```npm install```

Reply:
updated